GOALS.md

```diff
@@ -0,0 +1,71 @@
+
+What does this mean to the user experience?  We can build the entire system in less than a few minutes on the lowest spec hardware, and it will consume less than 50MB of storage.  This makes Gershwin ideal for it’s goal to run any hardware device using nearly any OS.  
+
+
+### Long term stable ABI
+
+* We do not want to fork or modify GNUstep core libs
+    * We have tickets open to address where we have made small modifications to core libs that will be ported back to GNUstep
+        * Such as examples include being able to restore applications by clicking a GNUstep app from the dock
+            * This was an example of a quick modification that was made to provide a proof of concept of Gershwin for technology previews
+            * Moving forward we will see whether it is feasible to move this functionality in Workspace instead or, make PR to put the behavior behind an option which is the standard way.
+
+We want to be able to provide multiple release versions of libs-base, libs-gui and so on in the future to ensure long term compatibility for applications built as GNUstep releases new versions.  Forking the core libs hinders our ability to do this properly as a project.
+
+### Consistent build systems
+
+* We must avoid pulling in external resources such as fonts, audio files, using tools outside of git
+* We currently typically publish overlays to gershwin-system but this may change in the future if we need to have a dedicated repo for fonts etc
+* The reasoning for this is FreeBSD packaging for example in works best when all resources are within our organization, and in our repos
+* We can ensure a consistent experience between packaging and building from source by not relying on outside tarballs, and cp -R methods that make things difficult for packagers.
+
+The goal here is to make keep gershwin-build simple, and also keep it simple for packagers to package our repos.  We want as many users using Gershwin as possible, and do not want to increase the level of effort for packagers to study gershwin-build to determine how to individually package each repo as needed.
+
+### Cloud & AI free
+
+It should go without saying that we  do not want any technology that would require user sign in, or anything that would violate a users privacy, reveal location, take automated screenshot of activities, provide automated reports of how long a user performs tasks on their devices.
+
+* We do not want to ship a software store that would require sign in or the user to identify whom they are
+* We do not want a Weather app that or installer that automatically determines a users location without the user requesting to do so
+* We do not want AI baked into any software Gershwin software
+    * It is of course okay to use copilot or other tools to review contributions, fix software we just don't want Gershwin software itself include AI features
```

Comment:
Would remove "we just don't want..." depending on the point above.

Reply:
Changing to cloud free, we can decide later the finer details in a follow up for AI.  I was mainly thinking we at least don't want to take screenshots of the users OS, etc.